app/lib/allowed_cross_namespace_usage.rb

```diff
@@ -7,5 +7,5 @@
 module AllowedCrossNamespaceUsage
   VendorAPIApplicationPresenter = VendorAPI::ApplicationPresenter
   RegisterAPISingleApplicationPresenter = RegisterAPI::SingleApplicationPresenter
-  VENDOR_API_VERSION = VendorAPI::VERSION
+  VENDOR_API_VERSION = VendorAPI::PRELIMINARY_VERSION
```

Comment:
do we still need this change?

Reply:
Yeso as we don't want the version rendered in support to change if we fix the version to 1.1 (as initially that will be used to make it available to the sandbox environment, and even when release in production it's unlikely that vendors will adopt it immediately). By then we should hopefuly have a better solution in place and enable switching between versions.